morib/src/main/java/org/morib/server/domain/category/application/DeleteCategoryServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package org.morib.server.domain.category.application;
+
+import lombok.RequiredArgsConstructor;
+import org.morib.server.domain.category.infra.Category;
+import org.morib.server.domain.category.infra.CategoryRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class DeleteCategoryServiceImpl implements DeleteCategoryService {
+    private final CategoryRepository categoryRepository;
+
+    @Override
+    public void deleteById(Long categoryId) {
+        Category category = categoryRepository.findById(categoryId).orElseThrow(
```

Comment:
오..! delete 관련 동작은 만약 해당 id값이 없거나 하면 무시해버리고 진행하는군요! 
이번에 배우고 갑니다!


// 내부로 파고 들어가서 발견
Deletes the entity with the given id.
If the entity is not found in the persistence store it is silently ignored.
매개변수:
id – must not be null.
던지기:
IllegalArgumentException – in case the given id is nul

Reply:
맞아요! deleteById가 결국 findById -> delete 순으로 진행하기때문에, findById 쪽에서 커스텀으로 예외 처리를 하기 위해 두개로 나누어 구현했습니다 ㅎㅎ